com.microsoft.mrtk.graphicstools.unity/Editor/Utilities/ShaderUtilities.cs

```diff
@@ -94,6 +99,14 @@ public static void GenerateCanvasMaterialAnimator(Shader shader, bool saveToPare
                 string name = ShaderUtil.GetPropertyName(shader, i);
                 string nameID = string.Format(PropertyID, name);
                 ShaderUtil.ShaderPropertyType type = ShaderUtil.GetPropertyType(shader, i);
+                string[] propertyAttributes = shader.GetPropertyAttributes(i);
+
+                string headerAttribute = Array.Find(propertyAttributes, isHeaderAttribute);
+                if (headerAttribute != null && headerAttribute != string.Empty)
```

Comment:
Super nit, string has a nice utility method for this check:
```suggestion
                if (!string.IsNullOrEmpty(headerAttribute))
```